tests/unit/Core/Content/ProductExport/Service/ProductExportGeneratorTest.php

```diff
@@ -159,6 +159,7 @@ private function getProductExportEntity(): ProductExportEntity
         $productExport->setSalesChannelId('salesChannelId');
         $productExport->setStorefrontSalesChannelId('storefrontSalesChannelId');
         $productExport->setProductStreamId('productStreamId');
+        $productExport->setBodyTemplate(null);
```

Comment:
are those setters needed (in the next two files as well), when the properties are nullable we should already declare them as null, so setting them explicitly to null shouldn't be needed 🤔

Reply:
yeah, because I didn't have a default null value for them before so the tests are failing, but it should be okay now, the setter won't hurt either way 🙈

Reply:
yeah should not be needed anymore, as null was added as default to all nullable fields

Reply:
I removed these setters